python/ray/tune/experimental/output.py

```diff
@@ -366,7 +372,7 @@ def _get_trial_table_data_per_status(
 def _get_trial_table_data(
     trials: List[Trial],
     param_keys: List[str],
-    metric_keys: List[str],
+    metric_keys: Union[List[str], Dict[str, str]],
```

Comment:
Same here, this can be an optional `Dict[str, str]` (instead of a list of str), in case user wants to map from actual keys to display names.

Reply:
Can we make the method just require a `Dict` instead? We only use this method in this one place.

Reply:
done